### PR TITLE
update(storage): update default minio resource limits and requests

### PR DESCRIPTION
**Pull request by soulseen**
Signed-off-by: soulseen <zhuxiaoyang1996@gmail.com>

### CODE REVIEW

File: pkg/cluster/controllers/storage/tenant.go [hunk 1]
```diff
@@ -247,19 +247,7 @@ func (m *MinIOController) getResourceRequirements(harborcluster *goharborv1.Harb
...
-	requests := map[corev1.ResourceName]resource.Quantity{
-		corev1.ResourceCPU:    resource.MustParse("250m"),
-		corev1.ResourceMemory: resource.MustParse("512Mi"),
-	}
-
-	return &corev1.ResourceRequirements{
-		Limits:   limits,
-		Requests: requests,
-	}
+	return &corev1.ResourceRequirements{}
```
Comment (heww) [inline pkg/cluster/controllers/storage/tenant.go:250]: Can we just return nil here?

Comment (soulseen) [reply pkg/cluster/controllers/storage/tenant.go:250]: I don't sure this is right, but return `&corev1.ResourceRequirements{}` must be correct。

Comment (soulseen) [reply pkg/cluster/controllers/storage/tenant.go]: looks like return `nil` works well,  and maybe it is a better choice, and to avoid memory escapes.

**Review by chlins — APPROVED** [review]
LGTM

**Review by bitsf — APPROVED** [review]

**Pull request merged**